recipes/cms/meta.yaml

```diff
@@ -0,0 +1,124 @@
+package:
+  name: cms
+  version: "2.0.0b"
+
+about:
+  home: http://broad-cms.readthedocs.io/en/latest/
+  license: GPLv3
+  summary: "CMS (composite of multiple signals) is a suite of tools for resolving signatures of recent positive selection in population genetic data."
+
+source:
+  fn: cms-2.0.0-beta.tar.gz
+  url: https://github.com/broadinstitute/cms/archive/2.0.0-beta.tar.gz
+  md5: 3e5754bf0a6eeb71cc483dcdae10d8c0
+
+build:
+  number: 0
+  rpaths:
+      - lib/
+      - lib//
+
+requirements:
+  build:
+    #- python
+    - gcc
+    - llvm
+
+  run:
+    - libgcc 
+    - alabaster ==0.7.8
+    - babel ==2.3.3
+    - bcftools ==1.3.1
+    - boltons ==16.4.1
+    - cosi2 ==2.02
+    - curl
+    - htslib ==1.3.1
+    - peewee ==2.8.0
+    - pysam ==0.9.1
+    - python-dateutil ==2.3
+    - samtools ==1.3.1
+    - selscan ==1.1.0b
+    - biopython ==1.67
+    - cloog ==0.18.0
+    - docutils ==0.12
+    - ecdsa ==0.13
+    - gcc ==4.8.5
+    - gmp ==5.1.2
+    - imagesize ==0.7.1
+    - isl ==0.12.2
+    - jinja2 ==2.8
+    - libgcc ==4.8.5
+    - matplotlib ==1.5.1
+    - markupsafe ==0.23
+    - mkl ==11.3.3
+    - mpc ==1.0.1
+    - mpfr ==3.1.2
+    - ncurses ==5.9
+    - openssl ==1.0.2h
+    - pandas ==0.18.1
+    - paramiko ==1.16.0
+    - pip ==8.1.2
+    - pycrypto ==2.6.1
+    - pygments ==2.1.3
+    - python ==3.5.1
+    - pytz ==2016.4
+    - readline ==6.2
+    - scipy ==0.17.1
+    - setuptools ==23.0.0
+    - six ==1.10.0
+    - snakemake
+    - snowballstemmer ==1.2.1
+    - sphinx_rtd_theme ==0.1.9
+    - sqlite ==3.13.0
+    - tk ==8.5.18
+    - wheel ==0.29.0
+    - xz ==5.0.5
+    - zlib ==1.2.8
+    - pysam ==0.9.1
+    - coverage ==4.1
+    - coveralls ==2.5.0 #?
+    - docopt ==0.6.2
+    - execnet ==1.3.0
+    - flake8 ==2.6.2
+    - mccabe ==0.5.0
+    - mock ==2.0.0
+    - pbr ==1.10.0
+    - peewee ==2.8.0
+    - py ==1.4.31
+    - pycodestyle ==2.0.0
+    - pyflakes ==1.2.3
+    - pytest ==3.0.3
+    - pytest-cov ==2.2.1
+    - pytest-xdist ==1.14
+    - python-dateutil ==2.3
+    - requests ==2.10.0
+    - sphinx ==1.4.8
+    - sphinx-argparse ==0.1.15
+    - sphinx_rtd_theme ==0.1.9
+    - apipkg ==1.4
+    - coveralls
```

Comment:
`coveralls` is available in the conda-forge channel as `python-coveralls`, so it can be renamed, and the recipe can be removed from this PR. Since this package is for distribution, coveralls is probably not needed at all (it's used by Travis to update coveralls.io)